js/stdlib/ts/ALGO.ts

```diff
@@ -2284,14 +2286,28 @@ const connectAccount = async (networkAccount: NetworkAccount): Promise<Account>
   const unsupportedAcc = stdAccount_unsupported(connector);
 
   const accObj = { ...unsupportedAcc, networkAccount, getAddress: selfAddress, 
-                   stdlib, setDebugLabel, tokenAccepted, tokenAccept, tokenMetadata, contract };
+                   stdlib, setDebugLabel, tokenAccepted, tokensAccepted: tokensAccepted_,
+                   tokenAccept, tokenMetadata, contract };
   const acc = accObj as unknown as Account;
   const balanceOf_ = (token?: Token): Promise<BigNumber> => balanceOf(acc, token);
   const balancesOf_ = (tokens: Array<Token | null>): Promise<Array<BigNumber>> => balancesOf(acc, tokens);
 
   return stdAccount({ ...accObj, balanceOf: balanceOf_, balancesOf: balancesOf_ });
 };
 
+const tokensAccepted = async (addr: Address): Promise<Array<Token>> => {
+  let assetHoldings: Array<AssetHolding>;
```

Comment:
Maybe do something like...
```suggestion
const tokensAccepted = async (addr_: Address): Promise<Array<Token>> => {
  const addr = protect(T_Address, addr_);
  let assetHoldings: Array<AssetHolding>;
```
?

Reply:
I think that balance does something like that